tests/integration/security/sds_ingress/util/util.go

```diff
@@ -269,6 +269,7 @@ type ExpectedResponse struct {
 	StatusCode                   int
 	SkipErrorMessageVerification bool
 	ErrorMessage                 string
+	AllowedErrorMessages         []string
```

Comment:
I am not sure we need to preserve backwards compatibility here but otherwise LGTM